content/terraform/v1.12.x/docs/language/block/module.mdx

```diff
@@ -490,27 +490,29 @@ The `for_each` argument is a meta-argument, which is built into Terraform and co
 - Default: None. 
 - Example: [Create multiple version of module resources](#create-multiple-versions-of-module-resources) 
 
-### `provider`
+### `providers`
 
-The `provider` argument instructs Terraform to use an alternate provider configuration. 
+The `providers` argument instructs Terraform to use an alternate provider configuration. 
 
 ```hcl
 module "<LABEL>" {
-  provider = <provider>.<alias>
+  providers = {
+   <provider> = <provider>.<alias>
+  }
 }
 ```
 
-By default, Terraform applies the default provider based on the module resource type, but you can create multiple provider configurations and use a non-default configuration for specific modules. 
+By default, the child module inherits the default provider configuration from its parent module, but you can create multiple provider configurations and use an alternate configuration for each module. Because a module is collection of different resources, some modules also let you specify a alternate configuration for different resources in the module. Refer to the module documentation for details.
```

Comment:
I think you meant to add a link too at the end of this sentence?
```suggestion
By default, a child module inherits the default provider configuration from its parent module, but you can create multiple provider configurations and use an alternate configuration for each module. Because a module is collection of different resources, some modules also let you specify an alternate configuration for different resources in the module. Refer to the module documentation for details.
```

Reply:
Also a small dumb nit is that modules inherit default provider configurations expect for the `source` and `version` - you probably cover that somewhere else, but figured I'd ask it just in case.